lib/maplibre-compose/src/maplibreNativeMain/kotlin/dev/sargunv/maplibrecompose/compose/offline/OfflineRegionManager.kt

```diff
@@ -0,0 +1,26 @@
+package dev.sargunv.maplibrecompose.compose.offline
+
+import androidx.compose.runtime.Composable
+
+@Composable public expect fun rememberOfflineRegionManager(): OfflineRegionManager
+
+public interface OfflineRegionManager {
+  public val regions: Set<OfflineRegion>
+
+  public suspend fun create(
+    definition: OfflineRegionDefinition,
+    metadata: ByteArray = ByteArray(0),
+  ): OfflineRegion
+
+  public suspend fun delete(region: OfflineRegion)
+
+  public suspend fun invalidate(region: OfflineRegion)
+
+  public suspend fun invalidateAmbientCache()
+
+  public suspend fun clearAmbientCache()
+
+  public suspend fun setMaximumAmbientCacheSize(size: Long)
+
+  public fun setOfflineMapboxTileCountLimit(limit: Long)
```

Comment:
```suggestion
  public fun setOfflineTileCountLimit(limit: Long)
```

I know, the standard format of vector tiles is **M**apBox **V**ector **T**iles, but the file format shouldn't matter, right? After all, MapLibre can also handle raster tiles.

Reply:
It's what the SDK calls it. I'm honestly unsure if the limit applies to all sources or if it's some vestigial remnant of Mapbox 

But yeah, best to rename it in our API I think